### ISSUE TITLE
[Spoil] bug? "hostility of the republic"

### ISSUE THREAD

**Issue by Oeleo** [endless-sky/endless-sky#264]
During the Free world mission's "The Free World's fleet has been scattered by a Navy Invasion" and I had to go in the Tarazed system. I didn't go here, I went to Kor Nor'peli to farm. My combat rating went from 11 to 12 and pouf, all the Republic ships and planets become friendly (to me) but Free World's and Republic's ships continued to fight together. Sorry for the long story but I don't really know what did that.

**Comment by endless-sky** [comment]
Your "reputation" with any government improves if you fight enemies of that government - e.g. pirates. So if you stop fighting for the Free Worlds and turn into a pirate hunter, sooner or later the Republic will decide you are no longer an enemy.

**Comment by Oeleo** [comment]
Oh ok, that's really nice but I take tribute from half of the republic's planet, so maybe that it will be more coherent if the limit of the hostility toward the player could be bigger (is it understable ?). Like that he really need to do an intentional "redemption" to change their attitude toward him.

**Comment by gchapyzh** [comment]
> Oh ok, that's really nice but I take tribute from half of the republic's planet.
Brother in arms! We are the same!

I do think that the factions forgive you a bit too easily. When I destroyed their gunboat and later a sensor drone, they all turned hostile, attacked me, but after I cleared the system of all ships and jumped, they became friendly again. AFTER I BLEW UP MULTIPLE CARRIERS THAT FIRED ON ME.

Perhaps have a special mission that will allow you to neutralize relationships with a faction.
